src/assets/scripts/utilities/unitConverters.js

```diff
@@ -220,30 +220,35 @@ export const heading_to_string = (heading) => {
  * @param coordinate {string}
  * @return transformedCoordinate {number}
  */
-export const parseCoordinate = (coordintae) => {
-    const match = REGEX.LAT_LONG.exec(coordintae);
+export const parseCoordinate = (coordinate) => {
+    const match = REGEX.LAT_LONG.exec(coordinate);
 
+    // If coordinate already in WGS84 ESPG:4326 form ('39.427618, -75.296011'), just return it as-is
     if (match == null) {
-        log(`Unable to parse coordinate ${coordintae}`);
-
-        return;
+        return coordinate;
     }
 
-    let transformedCoordinate = parseFloat(match[2]);
+    const degrees = parseFloat(match[2]);
+    let minutes = 0;
+    let seconds = 0;
 
+    // Gather minutes/seconds as decimal of a degree, if available
     if (match[5] != null) {
-        transformedCoordinate += parseFloat(match[5]) / 60;
+        minutes = (parseFloat(match[5]) * TIME.ONE_MINUTE_IN_HOURS);
```

Comment:
this certainly reads better.  One day I'd like to enumerate all the indicies this regex.match throws out.